packages/react-ui-validations/prod.tsconfig.json

```
@@ -1,5 +1,6 @@
 {
   "extends": "./tsconfig.json",
+  "include": ["./src/**/*", "./docs/**/*", "./typings/**/*"],
```

Можно еще и доку убрать, т.к. она собирается отдельно.

```suggestion
  "include": ["./src/**/*", "./typings/**/*"],
```